### DISCUSSION TITLE
IDEA编译0.15-rc04版，报错

### DISCUSSION THREAD

**Discussion by leafgodgood** [apache/doris#7295] [A - General / Q&A]
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.0.0:exec (gensrc) on project fe-core: Command execution failed.: Cannot run program "make" (in directory "G:\incubator-doris-0.15.0-rc04.tar\incubator-doris-0.15.0-rc04\incubator-doris-0.15.0-rc04\fe\fe-core"): CreateProcess error=2, 系统找不到指定的文件。 -> [Help 1]

之后做了这一步：通过拷贝linux上的 fe/fe-core/target/generated-sources 目录拷贝到相应的目录的方式实现

发现拷过去后又会被删掉：Deleting G:\incubator-doris-0.15.0-rc04.tar\incubator-doris-0.15.0-rc04\incubator-doris-0.15.0-rc04\fe\fe-core\target

0.15.0-rc04怎样本地编译调试fe

**Comment by SinyoWong** [comment]
请问这个问题解决了么？我也遇到了同样的情况。

**Reply by dataalive** [reply]
也是在win下IDEA里编译FE？报错栈一模一样吗？

**Reply by SinyoWong** [reply]
报错信息一模一样，我在win下编译的版本是最新发布的1.0.0

**Reply by caoliang-web** [reply]
通过拷贝linux上的 fe/fe-core/target/generated-sources 目录拷贝到相应的目录，不需要进行编译操作了，直接可以启动

**Reply by caoliang-web** [reply]
Cannot run program "make" 这个问题，需要安装MinGW,并在idea settings 里面设置Build,Execution,Deployment ->Build Tools -> Make 下面配置make 执行目录
